redux/slices/formBSlice.ts

```diff
@@ -53,10 +70,14 @@ export const loadFormBList = createAsyncThunk(
 export const loadSavedFormB = createAsyncThunk(
   "formB/fetchFormB",
   async (formId: string) => {
+    const state = store.getState();
+    // Grab the Covid flag status from the feature flags slice to use to deterimine if the Covid section should be displayed and if a default Covid object should be added to the form data
```

Comment:
Typo in "determine"

Reply:
I quite like deterimine. Defo belongs in the Covid section.